docs/section23/Rule23-11.md

```diff
@@ -0,0 +1,53 @@
+
+# Airside System - Rule 23-11 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** True  
+**Rule ID:** 23-11  
+**Rule Description:** System 11 Supply air temperature shall be reset from minimum supply air temp at 50% cooling load to room temp at 0% cooling load.    
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** G3.1.3.17 System 11 Supply Air Temperature and Fan Control  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-11.  
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. baseline_system_type_compare()
+
+**Applicability Checks:**  
+- create a list of the target system types: `APPLICABLE_SYS_TYPES = [HVAC_SYS.SYS_11]`
+- Get B-RMR system types: `baseline_system_types_dict = get_baseline_system_types(B-RMR)`
+
+- loop through the applicable system types: `for target_sys_type in APPLICABLE_SYS_TYPES:`
+    - and loop through the baseline_system_types_dict to check the system types of the baseline systems: `for system_type in baseline_system_types_dict:`
+        - do baseline_system_type_compare to determine whether the baseline_system_type is the applicable_system_type: `if((baseline_system_type_compare(system_type, target_sys_type, false)):`
+            - the systems in the list: baseline_system_types_dict[system_type] are sys-11 - loop through the list of systems: `for hvac_system_id in baseline_system_types_dict[system_type]:`
+            - `CONTINUE TO RULE LOGIC`
+            - otherwise, rule not applicable: `else: RULE_NOT_APPLICABLE`
+            
+  **Rule Logic:**
+  - create boolean for whether rule passes: `rule_passes = TRUE`
+  - get the hvac system: `hvac_system = get_object_by_id(hvac_system_id)`
+  - get the fan_system: `fan_system = hvac_system.fan_system`
+  - check whether the fan_system.temperature_control is LOAD_RESET_TO_SPACE_TEMPERATURE, other reset options do not result in correct control: `if fan_system.temperature_control == LOAD_RESET_TO_SPACE_TEMPERATURE:`
```

Comment:
Since this differs between heating and cooling mode, it does appear that we can check this.